plugins/GSdx/GSHwHack.cpp

```diff
@@ -2432,11 +2433,13 @@ void GSState::SetupCrcHack()
 {
 	GetSkipCount lut[CRC::TitleCount];
 
-	s_crc_hack_level = theApp.GetConfigI("crc_hack_level");
+	s_crc_hack_level = theApp.GetConfigT<CRCHackLevel>("crc_hack_level");
```

Comment:
Is `s_crc_hack_level` necessary? Unless I'm missing something, `m_crc_hack_level` could just be used.